### ISSUE TITLE
I can't use the last release.

### ISSUE THREAD

**Issue by Patriciooo** [psieg/Lightpack#520]
On M1 Mac mini always asking for microphone and screen. The only solution I have is restart from time machine local snap. ¿ Anyone with the same problem and solution ? Thanks.

**Comment by kimphuc0000** [comment]
> On M1 Mac mini always asking for microphone and screen. The only solution I have is restart from time machine local snap. ¿ Anyone with the same problem and solution ? Thanks.

Are you to grant permission Screen Recording on Security & Privacy -> Privacy -> Screen Recording -> Prismatik -> check the box and Microphone -> Prismatik -> check the box?

**Comment by kimphuc0000** [comment]
> 

Did Mac M warnng that 'The application “Prismatik” is not open anymore.' ?

**Comment by Patriciooo** [comment]
No. The app open.

**Comment by Kaitoukid93** [comment]
Try apple silicone version

**Comment by Patriciooo** [comment]
Yes I have a version from other issue open, but only work it from local snap copy, before are working fine, but no now. I know the last release it's universal, it and older apps don't work, only asking for grant camera and microphone in loop. Thanks.
